documentation/requirements.txt

```diff
@@ -1,8 +1,9 @@
-# Python dependencies to build the documentation
+# Python (PyPI) dependencies to build the documentation
 
-Sphinx ~= 3.0
+# Packages installed from conda
+# Sphinx ~= 3.0
```

Comment:
What happens if you leave these here, and remove them from the conda config?

Reply:
Then you get this readthedocs/readthedocs.org#6870